packages/core/src/Stores/notification-store.js

```diff
@@ -122,10 +123,7 @@ export default class NotificationStore extends BaseStore {
                     (Object.keys(root_store.client.account_status || {}).length > 0 &&
                         Object.keys(root_store.client.landing_companies || {}).length > 0)
                 ) {
-                    this.removeNotifications();
-                    this.removeAllNotificationMessages();
-                    this.setClientNotifications();
-                    this.handleClientNotifications();
+                    this.refreshNotifications();
```

Comment:
`refreshNotifications` function  calls `removeNotifications` with true for `should_close_persistent`.

But the previous implementation here had `removeNotifications` with false.

can you check this?